tests/adapters/switches/brocade_test.py

```diff
@@ -2146,38 +2166,61 @@ def test_remove_non_existent_dhcp_relay_server_fails(self):
-        "L2 protocols   : STP",
-        "Associated Virtual Interface Id: {}".format(vif_id),
-        "----------------------------------------------------------",
-        "No ports associated with VLAN",
-        "Arp Inspection: 0",
-        "DHCP Snooping: 0",
-        "IPv4 Multicast Snooping: Disabled",
-        "IPv6 Multicast Snooping: Disabled",
-        "Ve{} is down, line protocol is down".format(vif_id),
-        "  Type is Vlan (Vlan Id: {})".format(vlan_id),
-        "  Hardware is Virtual Ethernet, address is 748e.f8a7.1b01 (bia 748e.f8a7.1b01)",
-        "  No port name",
-        "  Vlan id: {}".format(vlan_id),
-        "  Internet address is 0.0.0.0/0, IP MTU 1500 bytes, encapsulation ethernet",
-        "  Configured BW 0 kbps",
-    ]
+    return VlanDisplayBuilder(vlan_id, name).with_vif_display(vif_id).to_strings()
+
+class VlanDisplayBuilder:
+    def __init__(self, vlan_id=9, vlan_name=""):
+        self.vlan_id = vlan_id
+        self.vlan_name = vlan_name
+        self.port_info = []
+        self.vif_id = None
+
+    def with_tagged_ports(self, ports=""):
+        return self.with_ports('Statically tagged Ports', ports)
+
+    def with_untagged_port(self, ports=""):
+        return self.with_ports('Untagged Ports', ports)
+
+    def with_ports(self, port_type="", ports=""):
+        self.port_info.append(
+            "{} : {}".format(port_type, ports)
+        )
+        return self
+
+    def with_vif_display(self, vif_id=None):
+        self.vif_id = vif_id
+        return self
+
+    def to_strings(self):
+        ret = [
+            "PORT-VLAN {}, Name {}, Priority Level -, Priority Force 0, Creation Type STATIC".format(self.vlan_id, self.vlan_name),
+            "Topo HW idx    : 81    Topo SW idx: 257    Topo next vlan: 0",
+            "L2 protocols   : STP",
+            ]
+        ret.extend(self.port_info)
+        ret.extend([
+            "Associated Virtual Interface Id: {}".format(self.vif_id or "NONE"),
+            "----------------------------------------------------------",
+            "No ports associated with VLAN",
+            "Arp Inspection: 0",
+            "DHCP Snooping: 0",
+            "IPv4 Multicast Snooping: Disabled",
+            "IPv6 Multicast Snooping: Disabled",
+        ])
+        if self.vif_id:
+            ret.extend([
+                "Ve{} is down, line protocol is down".format(self.vif_id),
+                "  Type is Vlan (Vlan Id: {})".format(self.vlan_id),
+                "  Hardware is Virtual Ethernet, address is 748e.f8a7.1b01 (bia 748e.f8a7.1b01)",
+                "  No port name",
+                "  Vlan id: {}".format(self.vlan_id),
+                "  Internet address is 0.0.0.0/0, IP MTU 1500 bytes, encapsulation ethernet",
+                "  Configured BW 0 kbps",
+            ])
+        else:
+            ret.append("No Virtual Interfaces configured for this vlan")
+        return ret
```

Comment:
Intellij -> setting -> editor -> general -> ensure lined feed at end of file on save